yugabyte/src/yugabyte/core.clj

```diff
@@ -0,0 +1,77 @@
+(ns yugabyte.core
+  (:require [clojure.tools.logging :refer :all]
+            [clojure.string :as str]
+            [jepsen [cli :as cli]
+                    [control :as c]
+                    [db :as db]
+                    [tests :as tests]
+                    [util :as util :refer [meh timeout]]
+            ]
+            [jepsen.control.util :as cu]
+            ))
+
+(def master-log-dir  "/home/yugabyte/master/logs")
+(def tserver-log-dir "/home/yugabyte/tserver/logs")
+
+(defn start!
+  "Starts YugaByteDB."
+  [node test]
+  (info node "Starting YugaByteDB")
+  (info (meh (c/exec (c/lit "if [[ -e /home/yugabyte/master/master.out ]]; then /home/yugabyte/bin/yb-server-ctl.sh master start; fi"))))
```

Comment:
Not clear how this will work for the first time. Guessing you run this on top of an already running universe?

Reply:
Yes, if a universe has been started at least once -  `/home/yugabyte/master/master.out` will be created on each master node and will remain there even after the universe is stopped and logs are removed.